### PR TITLE
feat(components/forms): tokenize toggle switch styles

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
[AB#3049570](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/3049570)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2865/storybook/)

Component Storybooks:

- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/colorpicker-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/errors-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/forms-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/indicators-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/lookup-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/phone-field-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/2865/storybooks/theme-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2865/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2865/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2865/playground/)

File: libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.scss [hunk 1]
```diff
@@ -1,157 +1,306 @@
...
+      );
+      border-radius: var(
+        --sky-override-toggle-switch-border-radius,
+        calc(
+            var(--sky-border-radius-thumb) +
+              calc(var(--sky-space-inset-thumb) * 2)
+          ) / 100%
+      );
+      flex: none;
+      line-height: $sky-line-height-base;
```
Comment (Blackbaud-ErikaMcVey) [inline libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.scss]: should we have a token here?

File: libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.scss [hunk 2]
```diff
@@ -1,157 +1,306 @@
...
+        calc(
+          calc(var(--sky-space-inset-thumb) * 2) - var(
+              --sky-border-width-input-focus
+            )
+        )
+      );
     }
 
-    &:not(:focus-visible, .sky-toggle-switch-disabled)
+    &:focus-visible:not(:is(:active, .sky-toggle-switch-disabled))
```
Comment (Blackbaud-ErikaMcVey) [inline libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.scss]: I think you can pass a comma separated list to `:not`, so `&:focus-visible:not(:active, .sky-toggle-switch-disabled)`

File: libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.scss [hunk 3]
```diff
@@ -1,157 +1,306 @@
...
+        background-color: var(
+          --sky-override-toggle-switch-background-color-selected,
+          var(--sky-color-background-selected-heavy)
+        );
+      }
 
-    &.sky-toggle-switch-disabled .sky-toggle-switch-switch {
-      cursor: not-allowed;
-    }
+      &:not(:is(:focus-visible, :active, .sky-toggle-switch-disabled))
```
Comment (Blackbaud-ErikaMcVey) [inline libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.scss]: same here, unless I'm misinterpreting these selectors

File: libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.scss [hunk 4]
```diff
@@ -1,157 +1,306 @@
...
+      .sky-toggle-switch-indicator {
+        left: 100%;
+      }
     }
 
-    &.sky-toggle-switch-checked .sky-toggle-switch-switch {
-      padding: 1px;
-    }
+    &.sky-toggle-switch-disabled {
+      color: var(--sky-text-color-default);
```
Comment (Blackbaud-ErikaMcVey) [inline libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.scss]: should be `--sky-color-text-default`

**Review by Blackbaud-ErikaMcVey — APPROVED** [review]

**Pull request merged**